test/integration/generated/bodyComplexWithTracing/src/operations/array.ts

```diff
@@ -44,25 +43,16 @@ export class ArrayImpl implements Array {
   async getValid(
     options?: ArrayGetValidOptionalParams
   ): Promise<ArrayGetValidResponse> {
-    const { span } = createSpan(
+    return tracingClient.withSpan(
       "BodyComplexWithTracing-getValid",
-      options || {}
+      options ?? {},
+      async () => {
+        return this.client.sendOperationRequest(
+          { options },
```

Comment:
```suggestion
          { updatedOptions },
```
I think, or is it `updatedOptions` ? `updatedOptions` in this case will be of type `OperationOptions` but not sure how that ties into here